scala/build/Dockerfile.maintenance

```
@@ -0,0 +1,263 @@
+  fonts-kiloji \
+  fonts-mikachan \
+  fonts-seto \
+  fonts-yozvox-yozfont \
+  fonts-aoyagi-kouzan-t \
+  fonts-aoyagi-soseki \
+  fonts-kouzan-mouhitsu
+#   ttf-mscorefonts-installer
+
+# Install Python 3.13 - v2024.2.40
+# Install Python 3.13 and required packages including distutils
+RUN apt-get update && apt-get install -y \
```

Hadolint

[info] Delete the apt-get lists after installing something

(DL3009)